src/main/java/org/opensearch/ad/model/AnomalyResultBucket.java

```diff
@@ -55,7 +55,7 @@ public static AnomalyResultBucket createAnomalyResultBucket(Bucket bucket) {
         return new AnomalyResultBucket(
             bucket.getKey(),
             (int) bucket.getDocCount(),
-            ((InternalMax) bucket.getAggregations().get(MAX_ANOMALY_GRADE_FIELD)).getValue()
+            ((ParsedMax) bucket.getAggregations().get(MAX_ANOMALY_GRADE_FIELD)).getValue()
         );
```

Comment:
This changed needs to be document right?

@joshpalis @owaiskazi19

Reply:
@vibrantvarun good call. Here you go: https://github.com/opensearch-project/opensearch-sdk-java/pull/717